git-server/start-up.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Is there a way to avoid this script and have the steps in the dockerfile?

Reply:
When the Dockerfile is interpreted (at image build time), the volume containing radicle work dir isn't mounted yet because volumes are mounted at container startup time (and not at image build time), so there's no place to copy the `pre-receive` hook to.   So the short answer is no I'm afraid.

Reply:
Note that the `bash` process only exists for a brief moment at container startup.  After it executes, it's `exec`'d into the actual git-server process.

Reply:
Ok got it, then I'd ask that we move this script a little bit out of the way, eg. `git-server/docker/start.sh` so it's obvious that it's meant to be used with docker.

Reply:
Sure.  Done ✅